libraries/AP_WayBack/AP_WayBack.cpp

```diff
@@ -0,0 +1,818 @@
+/*
+
+this library collects points of drone position, records them in memory and eliminates 
+unnecessary data - redundant points and loops, so it always knows 
+the shortest way to home via visited points.
+ 
+ Copyright night_ghost@ykoctpa.ru 2017
+ 
+ license GPLV3 or above
+   
+*/
+
+#include "AP_WayBack.h"
+
+extern const AP_HAL::HAL& hal;
+
+AP_WayBack::Point* AP_WayBack::points;
+uint16_t          AP_WayBack::max_num_points=0; // size of storage
```

Comment:
whoops, sorry. I thought these were in the class header. The ones in the header are fine. However, what are these variables doing here? They belong in the class as a member variable. What happens if there are two instances of the class?